packages/@webex/plugin-meetings/src/media/properties.ts

```diff
@@ -23,18 +24,18 @@ export type MediaDirection = {
  * @class MediaProperties
  */
 export default class MediaProperties {
-  audioTrack?: LocalMicrophoneTrack;
+  audioStream?: LocalMicrophoneStream;
   mediaDirection: MediaDirection;
   mediaSettings: any;
   webrtcMediaConnection: any;
-  remoteAudioTrack: any;
+  remoteAudioStream: RemoteStream;
   remoteQualityLevel: any;
-  remoteShare: any;
-  remoteVideoTrack: any;
-  shareVideoTrack?: LocalDisplayTrack;
-  shareAudioTrack?: LocalSystemAudioTrack;
+  remoteShare: RemoteStream;
```

Comment:
maybe this should be renamed to `remoteShareStream` for consistency?

Reply:
I agree. Actually, there's a `shareAudioStream` and a `shareVideoStream` already. We might as well remove it if its not used anywhere.

@brycetham - Any reason why we have this here?

Reply:
`shareAudioStream` and `shareVideoStream` are **local** webrtc-core streams, while `remoteShare` is for **remote** share stream - it's only used for transcoded meetings (for multistream we use receive slots and remote streams are managed by RemoteMediaManager), so because for now we don't support share audio in transcoded meetings, we only have 1 share remote stream - that's the remote video share